packages/cli-test/src/cli/shell.ts

```diff
@@ -203,6 +203,9 @@ export const shell = {
   },
   assembleShellEnv: function assembleShellEnv(): Record<string, string | undefined> {
     const spawnedEnv = { ...process.env };
+    if (process.platform === "win32"){
+      spawnedEnv.PATH = process.env.PATH;
```

Comment:
Without this the process couldn't find cmd.exe on the path

Reply:
lol windows